### ISSUE TITLE
Unhandled mnemonics

### ISSUE THREAD

**Issue by schrodyn** [danielplohmann/smda#48]
While bulk processing some files into MCRIT I witnessed some errors related to smda and unhandled mnemonics.

```
ERROR:smda.intel.IntelInstructionEscaper:********************************************** Unhandled mnemonic: endbr64
ERROR:smda.intel.IntelInstructionEscaper:********************************************** Unhandled mnemonic: vpsllw
ERROR:smda.intel.IntelInstructionEscaper:********************************************** Unhandled mnemonic: vcvtss2sd
```

Unfortunately I don't have hashes available as it was a bulk job of a large directory of samples. At least `endbr64` is a common instruction in ELF files compiled with GCC.

**Comment by danielplohmann** [comment]
Thanks for pointing this out! 
I have a list of some more known unhandled instructions locally and will try to fix as many of them as possible in one go, as this will affect the outcome of PicHash calculation and possibly lead to incompatibility and/or mismatches. 
Currently, the version of SMDA in MCRIT is already pinned to a commit prior to any changes that affect PicHash things, and whenever I will upgrade the list of instruction handling, I will do add a migration in MCRIT that will allow recalculation and updating of PicHashes for all functions to keep things straight.
The disassembly of SMDA itself however is not affected by these shortcomings in instruction handling at all.

**Comment by schrodyn** [comment]
Cheers, Daniel. Looking forward to the changes and thanks for your work on these projects.

**Comment by danielplohmann** [comment]
This has been fixed in smda v1.13.8. 
The escaper now covers all instructions known in capstone.
